pkg/ingester/ingester.go

```diff
@@ -178,6 +185,25 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
 	cfg.DefaultLimits.RegisterFlags(f)
 
 	f.StringVar(&cfg.IgnoreSeriesLimitForMetricNames, "ingester.ignore-series-limit-for-metric-names", "", "Comma-separated list of metric names, for which the -ingester.max-global-series-per-metric limit will be ignored. Does not affect the -ingester.max-global-series-per-user limit.")
+
+	f.BoolVar(&cfg.UtilizationLimitsEnabled, "ingester.utilization-limits-enabled", false, "Enable CPU/memory utilization based read path request limiting")
```

Comment:
This is not required. You can simply implicitly enable it when `-ingester.read-path-cpu-utilization-limit` or `-ingester.read-path-memory-utilization-limit` are > 0. This is common pattern in Mimir.

Reply:
Done.